engine/src/witness/arb.rs

```diff
@@ -49,26 +49,32 @@ where
 		+ 'static,
 	ProcessingFut: Future<Output = ()> + Send + 'static,
 {
-	let key_manager_address = state_chain_client
```

Comment:
I know this is not a valid change. I am having it here to highlight that we need to have a starting flow that works for the bouncer. I know @ramizhasan111 already discussed this with Alastair, please elaborate on what is the solution.

Reply:
tracked here: https://linear.app/chainflip/issue/PRO-1224/arbitrum-contracts-should-be-injected-on-runtime-upgrade

Reply:
We have the functionality to insert the contract addresses on runtime upgrade for live networks. For localnets we can have the contract addresses inserted on genesis via chainspec.

Reply:
That way the engine will read the correct addresses when starting up

Reply:
Yes agree. This is outdated and already removed. (just writing it here in case Dan sees these comments and gets confused).